Tweak course theme step text and images

alexsanford opened the pull request:
Fixes #6226

### Changes proposed in this Pull Request

* Tweak the text at the top of the step.
* Update the screenshots and HTML with the fixed text for "Course".

### Testing instructions

Load the Course Theme step in the Setup Wizard, and ensure the text and screenshots look correct.

### Note

Adding these screenshots is a tradeoff between size and sharpness. We converted them to the `webp` format, which helps, and downsized them a bit so that they aren't too huge. I allowed a bit higher quality for the main screenshot so everything above the fold looks pretty sharp (but it's also the biggest image). The lower screenshots are a bit lower quality.

The `sensei-lms.zip` package for version 4.9.0 is 10.6 MB. The package sizes for this PR are shown below.

### Packages

From commit 5545b128b (old, with low-res images): [sensei-lms.zip](https://github.com/Automattic/sensei/files/10188592/sensei-lms.zip) (12.1 MB)

From commit 27da18ab6 (high-res images): [sensei-lms.zip](https://github.com/Automattic/sensei/files/10197384/sensei-lms.zip) (13.7 MB)

alexsanford committed:
Fix text in onboarding step

alexsanford committed:
Change some wording

alexsanford committed:
Fix Course spelling

alexsanford committed:
Update images for course step

alexsanford committed:
Merge branch 'feature/onboarding-theme-step' into tweak/course-theme-step-text-images

jom commented on assets/setup-wizard/theme/index.js:
```diff
@@ -149,7 +149,7 @@ const Theme = () => {
 							onClick={ submitPage( false ) }
 						>
-							{ __( 'Skip theme selection', 'sensei-lms' ) }
+							{ __( 'Keep my current theme', 'sensei-lms' ) }
```
Nice change... I was wondering how we could make this clearer. I noticed we're still using `Skip` on the top bar when scrolling down, but it is fine saying `Skip` there.

jom reviewed (dismissed):
Looks good! Bit strange that it is the Setup Wizard E2E tests failing. I don't see what would make them fail. I've requested them re-run.

alexsanford commented:
Re-running the e2e tests again, but they seem to be failing every time. I'll dig into this.

alexsanford committed:
Fix e2e test for course theme step

alexsanford commented:
e2e tests are fixed in https://github.com/Automattic/sensei/pull/6251/commits/b7b80fc4f21e46e318dcb4097640660a6f79a286

Luchadores commented:
It's super nice, @alexsanford 🌟 

Here are a couple of notes while testing the theme:

- When the user scrolls to reveal the rest of the theme preview, the text at the top should scroll with the image at the same time. 
- The buttons appearing on the top right sticky bar are looking great!
- I noticed that the images of the theme are a bit blurry, although I did export them at retina size. Do let me know if you want to re-export them again.
- I also noticed we do not have hover states for the actions themselves. Do let me know if I need to provide you with these. For links for example, on hover, I would just remove the underline. 
- I also noticed on this PR test we don't have the image for the steps after the theme step.
- Mobile: Images are again blurry, we should have them exported at 2x. 
- Desktop: The images shouldn't increase in width with my screen. They should have the max size they have in Figma (similar to the other steps in the Wizard). Maybe that should sort out the business as well. 

- Bonus: As the user starts to scroll to explore the theme, or they click [Explore the theme], we should give the animation a feeling that they enter this exploration mode. What I mean here is a bit of an ease-in/out effect for the first scroll. Give the feeling that they enter this new mode where they explore the theme now. 

That's it! Awesome work!

@burtrw

alexsanford committed:
Fix scroll behaviour for course theme step

alexsanford committed:
Fix image widths

alexsanford committed:
Increased resolution of images

alexsanford commented:
@Luchadores I've addressed your above points:

> When the user scrolls to reveal the rest of the theme preview, the text at the top should scroll with the image at the same time.

This is fixed.

> I noticed that the images of the theme are a bit blurry, although I did export them at retina size. Do let me know if you want to re-export them again.

I've bumped up the resolution of the images. Note that this make the package even larger (13.7 MB now, as opposed to 12.1 MB with the lower-res images, and 10.6 MB before adding these images. This could have a negative effect on users with slow internet connections) cc @burtrw 

> I also noticed we do not have hover states for the actions themselves. Do let me know if I need to provide you with these. For links for example, on hover, I would just remove the underline.

This is the case for all buttons and links across the setup wizard, so I don't think I'll address it here. But feel free to create designs and add an issue for this 🙂 

> I also noticed on this PR test we don't have the image for the steps after the theme step.

Good catch! This is the case on the main feature branch as well. Created an issue to address this - https://github.com/Automattic/sensei/issues/6272

> Desktop: The images shouldn't increase in width with my screen. They should have the max size they have in Figma (similar to the other steps in the Wizard). Maybe that should sort out the business as well.

Fixed.

Luchadores commented:
Nice! Just tested it, thanks, @alexsanford!

jom reviewed (approved):
New changes look good!
